src/cell/mod.rs

```diff
@@ -0,0 +1,32 @@
+#[derive(Clone)]
+#[derive(Eq)]
+#[derive(PartialEq)]
+#[derive(Hash)]
+#[derive(Debug)]
```

Comment:
```suggestion
#[derive(Clone, Eq, PartialEq, Hash, Debug)]
```
Just syntax sugar, anyways I think `cargo fmt` does that automatically.

Reply:
Done!